RELEASING.md

```diff
@@ -0,0 +1,107 @@
+# Picard release process
+
+## Ensure git repository tree is clean
+
+From the local repository:
+
+### Check out the repo master branch
+
+```bash
+git fetch $PICARD_REMOTE && git checkout master
+```
+
+### Ensure nothing is on the way
+
+```bash
+git reset --hard $PICARD_REMOTE/master && git clean -f -d
+```
+
+#### Remove old compiled modules
+
+```bash
+find . -type f -name '*.pyc' -exec rm -f {} \;
+```
+
+### Tag to save the current state
+
+```bash
+git tag "before-release-$PICARD_VERSION" --force
+```
+
+### Remove any BOM nasty bytes from files
+
+This shouldn't be needed, but better to check before releasing
+
+```bash
+git ls-tree --full-tree -r HEAD --name-only |while read f; do sed -i '1s/^\xEF\xBB\xBF//' "$f"; done && git diff --quiet || git commit -a -m 'Remove nasty BOM bytes'
+```
+
+## Get latest translations from Transifex
+
+```bash
+python setup.py get_po_files && git diff --quiet || git commit -m 'Update .po files' -- po/
+```
+
+## Synchronize generated consts
+
+```bash
+python setup.py update_constants && git diff --quiet || git commit -a -m 'Update constants' -- picard/const/*.py
+```
+
+## Synchronize picard.pot with sources
+
+```bash
+python setup.py regen_pot_file && git diff --quiet || git commit -m 'Update pot file' -- po/picard.pot
+```
```

Comment:
This is something that should be done from time to time, espcially some time before the release. Doing this at actual release time is nice, but kind of pointless. Should we mention that somewhere at the top of the file as release preparation?